Sources/XcodeSupport/XcodeProject.swift

```diff
@@ -63,8 +64,34 @@ final class XcodeProject: XcodeProjectlike {
         }
 
         targets = Set(xcodeProject.pbxproj.nativeTargets
-            .map { XcodeTarget.make(project: self, target: $0) }
+            .map { XcodeTarget(project: self, target: $0) }
             + subProjects.flatMap { $0.targets })
+
+        let shouldLoadPackages = targets.contains { $0.hasPackageProductDependencies }
+
+        if shouldLoadPackages {
+            // The project file does not provide a clear way to identify file references for local SPM packages.
+            // We need to iterate over all references and check for folders containing a Package.swift file.
+            for fileRef in xcodeProject.pbxproj.fileReferences {
+                // To avoid checking every single file reference, narrow our search down to the known file types Xcode uses
+                // for package references.
+                let lastKnownFileType = fileRef.lastKnownFileType
+
+                guard (lastKnownFileType == "text" || lastKnownFileType == "wrapper"),
```

Comment:
In testing this on my project, I needed to add a case for `lastKnownFileType == "wrapper"` here.